new_tab/about.html

```diff
@@ -0,0 +1,159 @@
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <main>
+      <div id="customBackground" class="customBackground">
+        <div class="aboutPage">
+          <h2>About Us</h2>
+        </div>
+        <p class="aboutPara">
+          The Global Leaders' Conference brings together a community of expert,
+          intellectuals and scholars who make up the Global leaders' fratenity
+          affiliate network in a diffrent country every two years. This year we
+          hope to expand our scope across the entire Microverse reaching out to
+          the globe. Lorem ipsum dolor sit amet consectetur adipisicing elit.
+          Perferendis consequatur, ratione quod omnis ipsa aperiam quaerat
+          facilis recusandae quas hic saepe dolore a debitis, quos inventore
+          commodi nihil reiciendis consectetur!
+        </p>
+
+        <div class="contactSec">
+          <p>
+            Please contact us per Email for any further questions. We would
+            really be glad to assist you! <br />
+            <u> <b>globaleadercon@yahoo.com.</b> </u> <br />
+            or please call +1354 26569 554
+          </p>
+        </div>
+
+        <div class="News">
+          <h3>
+            News Updates <br />
+            Global Leaders' Conference
+          </h3>
+          <p>
+            GLCon23' <ion-icon id="naveicon" name="aperture-outline"></ion-icon>
+          </p>
+        </div>
+
+        <h4 id="Location" ><u>See Past and Future Events! </u></h4>
+        <span>
+          Take a quick glance @ the latest Conference held in Ghana, Accra
+        </span>
+
+      </div>
+      
+      <div class="eventGallary">
+        <a href="#" class="imageArea1">
+          <p>
+            <b>2023</b> <br>
+            Global Leaders Conference in Ghana, Accra.
+          </p>
+
+        </a>
+        <a href="#" class="imageArea2">
+          <p>
+            <b>2023</b> <br>
+            Global Leaders Conference in Ghana, Accra.
+          </p>
+
+        </a>
+      </div>
+
+      <section>
+        <div id="sponsors" class="Partners">
+          <h2>Parnters</h2>
+          <span id="partdash">__</span>
+          <div class="logoContainer">
+            <img src="../Images/apple.png" alt="Apple Logo" />
+            <img src="../Images/google.png" alt="Google" />
+            <img src="../Images/mozilla.png" alt="mozilla" />
+            <img src="../Images/tesla.png" alt="Tesla" />
+          </div>
+        </div>
+      </section>
+    </section>
+
+
+    </main>
+
+    <footer>
+      <div class="footerContainer">
+        <div class="foot">
+          <span
+            >GLCon23'<ion-icon id="naveicon1" name="aperture-outline"></ion-icon
+          ></span>
+        </div>
+        <div id="footer-date">
+          2023 Global Leaders Conference <br />
+          USA All Rights Reserved
+        </div>
+      </div>
+    </footer>

```

Comment:
-  [ ] As I mentioned above :point_up_2: please stick to the Behance design but the background color of your footer is not the same as the Behance design so kindly as the correct background color which is match with Behance design :+1: check below :point_down:  the comparison between your design and Behance design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-07-07 15-59-16](https://github.com/Mike47ip/JavascriptCapstone/assets/45226848/61780e98-f69c-46f4-bd83-7f1b43a328a2)

**Behance Design**
![Screenshot from 2023-07-07 15-59-30](https://github.com/Mike47ip/JavascriptCapstone/assets/45226848/fb629aa5-4321-4e98-b236-8bce1ebb914e)